roachtest: schemachange/indexrollback/tpcc-1000 failed

SHA: https://github.com/cockroachdb/cockroach/commits/3f22b976765a7cc58b97a9fd9866f2f26493c987

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=schemachange/indexrollback/tpcc-1000 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1097785&tab=buildLog

```
The test failed on release-2.1:
	test.go:1118: test timed out (1h0m0s)
	test.go:713,cluster.go:1164,tpcc.go:120,cluster.go:1490,errgroup.go:57: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1097785-schemachange-indexrollback-tpcc-1000:5 -- ./workload run tpcc --warehouses=1000 --histograms=logs/stats.json --wait=false --tolerate-errors --ramp=5m0s --duration=30m0s {pgurl:1-4} returned:
		stderr:
		
		stdout:
		     2.0            1.3   4026.5   7247.8   7247.8   7247.8 newOrder
		    1m0s      185            0.0            0.4      0.0      0.0      0.0      0.0 orderStatus
		    1m0s      185            0.0            0.1      0.0      0.0      0.0      0.0 payment
		    1m0s      185            1.0            0.4    113.2    113.2    113.2    113.2 stockLevel
		E190117 13:56:15.534047 1 workload/cli/run.go:402  error in payment: ERROR: context deadline exceeded (SQLSTATE XX000)
		_elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
		    1m1s      187            0.0            0.4      0.0      0.0      0.0      0.0 delivery
		    1m1s      187            3.0            1.3  12348.0  53687.1  53687.1  53687.1 newOrder
		    1m1s      187            0.0            0.4      0.0      0.0      0.0      0.0 orderStatus
		    1m1s      187            0.0            0.1      0.0      0.0      0.0      0.0 payment
		    1m1s      187            0.0            0.4      0.0      0.0      0.0      0.0 stockLevel
		: signal: killed
	test.go:713,cluster.go:1511,tpcc.go:130,schemachange.go:375: timeout with current status: "running"

```

--- a/pkg/cmd/roachtest/schemachange.go
+++ b/pkg/cmd/roachtest/schemachange.go
@@ -325,7 +325,7 @@ func makeIndexAddTpccTest(numNodes, warehouses int, length time.Duration) testSp
 }
 
 func registerSchemaChangeCancelIndexTPCC1000(r *registry) {
-	r.Add(makeIndexAddRollbackTpccTest(5, 1000, time.Minute*30))
+	r.Add(makeIndexAddRollbackTpccTest(5, 1000, time.Minute*90))
 }
 
 // Creates an index and job, returning the job ID and a notify channel for

